opening_hours/decorators.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+
+from django.utils.timezone import get_default_timezone
+
+TIMEZONE = get_default_timezone()
+
+
+def datetime_args_to_default_timezone(func: callable):
+    def convert_to_utc(*args, **kwargs):
+        converted_args = []
+        for arg in args:
+            if isinstance(arg, datetime.datetime):
+                arg = arg.astimezone(TIMEZONE)
+            elif isinstance(arg, datetime.time):
+                timezone = arg.tzinfo
+                if not timezone:
+                    timezone = TIMEZONE
+                dt = timezone.localize(
+                    datetime.datetime(1977, 1, 1, arg.hour, arg.minute)
```

Comment:
Seconds are lost when the time passes through this function.

I think it probably doesn't matter much in our use cases since I think we're not really dealing with one-second granularity. But we could add `arg.second` here for completeness. What do you think?

Reply:
Yep I agree, those seconds would be good to have _just_in_case_

Reply:
Fixed this.